biggraphite/cli/command_clean.py

```diff
@@ -72,6 +72,14 @@ def add_arguments(self, parser):
             default="",
             action="store"
         )
+        parser.add_argument(
+            "--num-token-ignore-on-error",
+            help="Specify the number to token to pass to continue cleaning"
```

Comment:
The default is 0.001 not 0.005 right ? The value is passed as a number not a percentage ?
Also there is a typo `Specify the number of tokens`